README.md

```diff
@@ -112,42 +112,36 @@ The main class for interacting with the Cohort Experience Space.
-| **spinnerStyle?.backgroundColor** | `#E8E9E8`                                                  | The background color of the spinner                                                                                                                         |
-| **urlParams?.disableLogout**      | `true`                                                     | Whether to display the logout button in the Experience Space                                                                                                |
-| **urlParams?.navbar**             | `true`                                                     | Whether to display the navbar in the Experience Space                                                                                                       |
-| **urlParams?.navigationType**     | `tabbar`                                                   | The navigation type of the Experience Space for mobile devices. Possible values: `tabbar` or `burger`                                                       |
+> - `cohort` (Default): The session of the user is managed by Cohort. The user will be required to login and logout inside the iframe using their email address. This is the simplest authentication mode. Use this if you don't have an existing user authentication system, or if you don't want to integrate Cohort with it. Sessions will be cached automatically inside the ifram (so the user won't need to sign-in again if they have signed-in in the past on the same browser).
+> - `custom`: You are in charge of managing the session of the user. You must provide the following parameters:
+>   - `userEmail`: The email of the user currently logged-in in your application (or null, if the user is logged out)
+>   - `getAuthToken`: A function that you must implement to retrieve the Cohort authToken for the currently logged-in user using the Cohort Merchants API. This function will be called by the SDK when we need an authentication token to sign the user in the iframe.
+>   - `customLoginUrl`: The URL of your application authentication page. If you allow logged-out users to see the Experience Space, they will be redirected to this URL when they try to access a protected page. The URL of the page the user must be redirected to after login will be passed as a parameter. For the best user experience, we recommend that your authentication page supports it and redirects the user to it after login.
+>   - `customLoginRedirectParameterName`: The name of the URL parameter that will contain the URL to redirect the user after login. By default, it is `destination`.
+>     With the `custom` auth mode, the SDK will take care of knowing when the user is authenticated inside the iframe and will only request an auth-token when needed, so there is no need for you to implement any session caching.
+
+You can also specify a list of options for the iframe:
```

Comment:
You've forgotten to update the whole tab here, parameters are no longer like that